.prettierrc.js

```diff
@@ -0,0 +1,10 @@
+'use strict';
+
+// https://prettier.io/docs/en/options.html
+module.exports = {
+	arrowParens: 'always',
+	printWidth: 100,
+	singleQuote: true,
+	trailingComma: 'es5',
```

Comment:
What about the "all" option? The minimal supported Node.js version is `8.7.0`, so we can safely use it.

Reply:
`all` is fine by me.